keras/engine/training.py

```diff
@@ -1922,13 +1923,12 @@ def fit_generator(self, generator,
         using `use_multiprocessing=True`.
 
         # Arguments
-            generator: A generator or an instance of Sequence (keras.utils.Sequence)
+            generator: A generator or an instance of `Sequence` (`keras.utils.Sequence`)
                     object in order to avoid duplicate data
                     when using multiprocessing.
                 The output of the generator must be either
-                - a tuple (inputs, targets)
-                - a tuple (inputs, targets, sample_weights).
-                All arrays should contain the same number of samples.
```

Comment:
arrays don't have to contain the same number of samples. In fact, most commonly the last one won't.